engine/runtime/src/main/java/org/enso/interpreter/runtime/Module.java

```diff
@@ -680,7 +739,8 @@ boolean isMemberInvocable(String member) {
         || member.equals(MethodNames.Module.SET_SOURCE)
         || member.equals(MethodNames.Module.SET_SOURCE_FILE)
         || member.equals(MethodNames.Module.GET_ASSOCIATED_TYPE)
-        || member.equals(MethodNames.Module.EVAL_EXPRESSION);
+        || member.equals(MethodNames.Module.EVAL_EXPRESSION)
+        || member.equals(MethodNames.Module.EVAL_EXPRESSION_LOCAL_SCOPE);
```

Comment:
I'd rather implement this without special `EVAL_EXPRESSION_LOCAL_SCOPE` message.